### PR TITLE
[DO NOT MERGE] BXMSPROD-598 business-central-webapp and business-monitoring-webapp splitted in two

### PR DESCRIPTION
**Pull request by Ginxo**
BXMSPROD-598

### CODE REVIEW

File: business-central-parent/add-ons-distribution/target/antrun/build-main.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project name="maven-antrun-" default="main"  >
+<target name="main">
+  <copy includeemptydirs="false" todir="/home/emingora/development/projects/RedHat/issues/BXMSPROD-598/kie-wb-distributions/business-central-parent/add-ons-distribution/target/classes" failonerror="false" quiet="true">
```
Comment (manstis) [inline business-central-parent/add-ons-distribution/target/antrun/build-main.xml]: Was this file left by mistake?

If not, I suspect the paths are wrong.

Comment (Ginxo) [reply business-central-parent/add-ons-distribution/target/antrun/build-main.xml]: Sure, .gitignore file added to add-ons-distribution module and target folder removed. Thanks @manstis

**Comment by Ginxo** [comment]
do not merge it yet, please. Don't spend time on this for the moment. We still working with thorntail team in order to fix one pending thing. 
It would be nice to have a label "do_not_merge" like the one we have in droolsjbpm-build-bootstrap https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1141 :thinking:

**Comment by manstis** [comment]
@Ginxo label added.

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Comment by Ginxo** [comment]
Closed since https://github.com/thorntail/thorntail/pull/1362 has been merged

**Pull request closed without merging**